design-library/source/scss/base/_variables.scss

```diff
@@ -0,0 +1,40 @@
+
+$base-font-family: "Source Sans Pro", "Helvetica Neue", "Helvetica", "Roboto", "Arial", sans-serif;
```

Comment:
Are these variables and styles based on my proposed design or the USWDS work?

Reply:
"Source Sans Pro" and "Merriweather" are purely in the interest of USWDS - those are the Google fonts they specify.